docs/docs/resources/pipeline-components/kafka-app.yaml

```diff
@@ -0,0 +1,48 @@
+# Base component for Kafka-based components.
+# Producer or streaming apps should inherit from this class.
+- type: kafka-app # required
+  name: kafka-app # required
+  namespace: namespace # required
+  # `app` can contain application-specific settings, hence  the user is free to
+  # add the key-value pairs they need. The fields that are specific to kafka
+  # apps are marked with `# kafka-app-specific`
+  app: # required
+    streams: # required, kafka-app-specific
+      brokers: ${broker} # required
+      schemaRegistryUrl: ${schema_registry_url}
+    nameOverride: override-with-this-name # kafka-app-specific
+    key1:
+      key4: value3
+    key2: value1
+    key3: value2
```

Comment:
what is the meaning of those values?

Reply:
There is no meaning, they are there to show that you could put anything there, but I think probably it's better to instead add some real configs from streams-bootstrap that work for both the producer and the streams-app

I've also ditched the weird `key: value` lines from `kubernetes-app`